web/src/sidebar_ui.ts

```diff
@@ -6,6 +6,7 @@ import render_right_sidebar from "../templates/right_sidebar.hbs";
 
 import {buddy_list} from "./buddy_list.ts";
 import * as channel from "./channel.ts";
+import {maybe_show_scrolling_formatting_buttons} from "./compose_ui.ts";
 import {media_breakpoints_num} from "./css_variables.ts";
 import {reorder_left_sidebar_navigation_list} from "./left_sidebar_navigation_area.ts";
 import {localstorage} from "./localstorage.ts";
```

Comment:
This file doesn't use our preferred `import * as` style; functions should always be imported so that they're called using the module name, for easier grepping. My guess is it is the result of cargo-culting from a few modules things that are usually handled differently, like `buddy_list`.

Reply:
Yes, or just cargo-culting from the lines below :) Happy to fix.